.github/workflows/run-testplans.yml

```diff
@@ -7,6 +7,24 @@ on:
         description: the directory with the testplans test
         required: true
         type: string
+      extra-versions:
+        description: artifact name for the extra-versions.json file
+        required: false
+        type: string
+      image-tar:
+        description: artifact name for the image.tar(s) to import
+        required: false
+        type: string
```

Comment:
Similar to my comment on the go-libp2p PR, could this be merged into a single option if we make use of tags?

Reply:
No. This was a misunderstanding.